frontend/src/components/form/inputs/Interpreter.tsx

```diff
@@ -42,89 +60,182 @@ const StyledField = ({ name, label, rows = { xs: 6 } }: GridItemInputProps) => (
     </StyledFormControl>
     <FieldError name={name} />
   </Grid>
-)
+);
+
+const StyledDateField = ({
+  name,
+  label,
+  rows = { xs: 6 },
+  initialValue,
+}: GridItemInputProps) => {
+  const [date, setDate] = React.useState('');
+  const [, , setField] = useField(name);
```

Comment:
Don't like the name `setField` here, possibly you could pattern match it out like:
```
const [, , { setValue }] = useField(name);
```